### DISCUSSION TITLE
C4 Model Diagrams in Excalidraw

### DISCUSSION THREAD

**Discussion by symplrdudley** [excalidraw/excalidraw#8764] [General]
Hello Excalidraw community!

I was wanting to know if anyone had ever successfully used Excalidraw to make a C4 Model diagram? I was looking at Mermaid and saw the C4 Diagram. 

- [The C4 model for visualising software architecture | C4 Model (c4model.com)](https://c4model.com/).
- [C4 Model | Wikipedia (wikipedia.org)](https://en.wikipedia.org/wiki/C4_model).

**Comment by Mrazator** [comment]
> I was wanting to know if anyone had ever successfully used Excalidraw to make a C4 Model diagram?

There shouldn't be anything super specific about a C4 that wouldn't be possible to do in Excalidraw. In fact, there is even a C4 library for bootstrapping.

<img width="879" alt="Screenshot 2024-11-07 at 12 55 25" src="https://github.com/user-attachments/assets/b7196604-2b49-4137-b99a-4c786ccdae2d">
<img width="1920" alt="Screenshot 2024-11-07 at 12 55 59" src="https://github.com/user-attachments/assets/24863a12-ebe4-4552-abd6-6c9c1cb58f79">
